docs/api/version-history.md

```diff
@@ -23,6 +23,7 @@ keywords: "API, Docker, rcli, REST, documentation"
   If `Error` is `null`, container removal has succeeded, otherwise
   the test of an error message indicating why container removal has failed
   is available from `Error.Message` field.
+* `GET /containers/(name)/logs` now supports an additional query parameter: `until`, which returns log lines that occurred before the specified timestamp.
```

Comment:
DOH! Forgot about this again; this should now be under v1.35; I'll do a quick follow up 😅

Reply:
opened https://github.com/moby/moby/pull/35398 for that